src/main/java/org/sonarlint/intellij/notifications/SonarLintProjectNotifications.java

```diff
@@ -133,4 +140,34 @@ public void expireCurrentHotspotNotificationIfNeeded() {
       currentOpenHotspotNotification = null;
     }
   }
+
+  public void handle(ShowSmartNotificationParams smartNotificationParams) {
+    var telemetry = getService(SonarLintTelemetry.class);
+    var category = smartNotificationParams.getCategory();
+    telemetry.devNotificationsReceived(category);
```

Comment:
Isn't this managed by the backend already ?

Reply:
Yes you're right, only dev notifications clicked should be counted on client side